go.mod

```diff
@@ -6,6 +6,7 @@ go 1.13
 replace maze.io/x/crypto => github.com/snapcore/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
 
 require (
+	github.com/alexkohler/nakedret v1.0.1 // indirect
```

Comment:
why does it end up here now? it's a not a dep of the code itself

Reply:
It happened to me too during tests. I suspect that it's because he ran "go get -u github.com/alexkohler/nakedret@v1.0.1" to do a test, and it added it to the repository.

Reply:
Yes, it's added by `go get` when running the script. Unfortunately, running it in GOPATH mode doesn't support the path@version syntax